examples/app/log_layers_ecs.rs

```diff
@@ -23,17 +23,17 @@ use bevy::{
 
 /// A basic message. This is what we will be sending from the [`CaptureLayer`] to [`CapturedLogEvents`] non-send resource.
 #[derive(Debug, Event)]
-struct LogEvent {
+pub struct LogEvent {
```

Comment:
What is your reasoning for making these structures public? Examples are not libraries, so no other code should be relying on these types.

Reply:
I added it, because doc comments need public access to enable linking to items. But you are right I think I will will remove the `intra-doc links` that are causing problem with ci and keep symbols in private to keep consistency with the other examples.